R/dates_secuTrial.R

```diff
@@ -20,6 +20,7 @@ dates_secuTrial <- function(x, ...) UseMethod("dates_secuTrial", x)
 #' @rdname dates_secuTrial
 #' @export
 dates_secuTrial.secuTrialdata <- function(object){
+  if (object$export_options$dated) warning("dates already added")
```

Comment:
@aghaynes would it make sense to stop the function here is dated is TRUE? I don't see any reason to run it again.

Reply:
that's a fair enough point, although stop results in an error. Maybe someone has some reason to rerun it (e.g. they changed the date format detected by load_export_options). I don't know... I mostly wanted to avoid errors I think